vercel.json

```diff
@@ -1,7 +0,0 @@
-{
-  "build": {
-    "env": {
-      "ENABLE_FILE_SYSTEM_API": "1"
-    }
-  }
-}
```

Comment:
Got this message in vercel
<img width="1214" alt="CleanShot 2022-07-02 at 19 53 40@2x" src="https://user-images.githubusercontent.com/1051509/177011304-a46d1697-fb1b-41f2-8d75-515ced78f1cf.png">